src/app/static/src/tests/components/choroplethFilter.test.ts

```diff
@@ -1,6 +1,6 @@
 import {createLocalVue, shallowMount} from '@vue/test-utils';
 import Vue from 'vue';
-import Vuex from 'vuex';
```

Comment:
Changes to this file are irrelevant to this ticket but I was seeing errors when running tests saying things like "action choroplethFilterUpdated not defined"/"mutation ChoroplethFilterUpdated not defined" so I just tidied this up to fix those.